packages/bot-strategy-runner/src/index.ts

```diff
@@ -0,0 +1,213 @@
+import program from "commander";
+import nodeFetch from "node-fetch";
+import assert from "assert";
+import * as fs from "fs";
+import * as cliProgress from "cli-progress";
+import * as bluebird from "bluebird";
+
+import { Logger, delay } from "@uma/financial-templates-lib";
+
+import { strategyRunnerConfig, buildBotConfigs, buildGlobalWhitelist, mergeConfig } from "./ConfigBuilder";
+import { runBot } from "./BotEntryWrapper";
+
+// Defaults for bot execution. If the user does not define these then these settings will be used.
+const defaultPollingDelay = 300;
+const defaultStrategyTimeout = 120;
+const defaultBotConcurrency = 10;
+const defaultNetwork = "mainnet_mnemonic";
+
+// Global progress bar to show the status of the strategy runner during it's execution.
+let counter = 0;
+const progressBar = new cliProgress.SingleBar(
+  { format: "[{bar}] {percentage}% | bots executed: {value}/{total} | recent execution: {botIdentifier}" },
+  cliProgress.Presets.shades_classic
+);
+
+// Main entry point that takes in a strategyRunnerConfig and executes bots within an execution while loop.
+async function runStrategies(strategyRunnerConfig: strategyRunnerConfig) {
+  strategyRunnerConfig = _setConfigDefaults(strategyRunnerConfig); // Default important configs. User configs take preference.
+  Logger.debug({
+    at: "BotStrategyRunner",
+    message: "Starting bot strategy runner",
+    strategyRunnerConfig: strategyRunnerConfig
+  });
+
+  // Generate a global whitelist of addresses that all enabled bots will run on.
+  const globalWhiteList = await buildGlobalWhitelist(strategyRunnerConfig);
+
+  // Construct bot configs for all enabled bots. This includes all bot specific overrides, whitelists, blacklists and any
+  // extra settings defined in the config. See the readme for full details on what is possible with this config.
+  const allBotsConfigs = await buildBotConfigs(globalWhiteList, strategyRunnerConfig);
+  console.log("allBotsConfigs", allBotsConfigs);
```

Comment:
Is this console.log supposed to be here?

Reply:
nope. have remove it!